subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/integration/KotlinScriptCompilerTest.kt

```diff
@@ -68,13 +75,7 @@ class KotlinScriptCompilerTest : TestWithTempFiles() {
         }
     }
 
-    open class FooTemplate(
```

Comment:
❓ IIRC, the template had to be moved out due to a bug in the compiler, right?

It would be nice to keep a reference to the YT issue, if any, in a comment.